op-e2e/system/e2esys/setup.go

```diff
@@ -904,6 +907,11 @@ func (cfg SystemConfig) Start(t *testing.T, startOpts ...StartOption) (*System,
 		DataAvailabilityType:  sys.Cfg.DataAvailabilityType,
 		CompressionAlgo:       compressionAlgo,
 		AltDA:                 batcherAltDACLIConfig,
+
+		ThrottleThreshold: cfg.ThrottleThreshold,
+		ThrottleInterval:  500 * time.Millisecond,
+		ThrottleTxSize:    cfg.ThrottleTxSize,
+		ThrottleBlockSize: cfg.ThrottleBlockSize,
```

Comment:
maybe we usually disable this (via interval==0) in tests and only enable it via some `SystemOpt`?

Reply:
I see, if I use interval==0 instead of throttle-threshold==0 indicate "never throttle" that could clean things up a bit...looking into it....

Reply:
Unfortunately the StartOptions there only seem to allow modifiers that act on the System & SystemConfig structs, but not the Batcher CLIConfig.

Reply:
I pushed a commit to improve modular batcher configuration in e2e tests, and always disabled batcher throttling in e2e tests, except in your throttling test.